main.tf

```diff
@@ -0,0 +1,57 @@
+locals {
+  path_to_binary = "${var.file_path}/${var.file_name}.zip"
+}
+
+resource "aws_cloudwatch_log_group" "lambda_s3" {
+  name              = "/aws/lambda/${var.environment}_${var.file_name}"
```

Comment:
What happens if `var.environment` is capital?

Reply:
Have added validation check for env to be lowercase word, also extracted this out to be a local